src/main/java/com/ajou/hertz/domain/administrative_area/entity/Sido.java

```diff
@@ -0,0 +1,32 @@
+package com.ajou.hertz.domain.administrative_area.entity;
+
+import com.ajou.hertz.common.entity.TimeTrackedBaseEntity;
+import com.ajou.hertz.domain.administrative_area.constant.SidoList;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.EnumType;
+import jakarta.persistence.Enumerated;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import lombok.AccessLevel;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+@Getter
+@AllArgsConstructor(access = AccessLevel.PRIVATE)
+@NoArgsConstructor(access = AccessLevel.PROTECTED)
+@Entity
+@Table(name = "adminitrative_area_sido")
+public class Sido extends TimeTrackedBaseEntity{
+
+	@Id
+	@Column(nullable = false, updatable = false)
+	private Long id;
```

Comment:
```suggestion
	@Id
	@GeneratedValue(strategy = GenerationType.IDENTITY)
	@Column(name="administrative_area_sido_id", nullable = false, updatable = false)
	private Long id;
```

1. PK 생성 전략은 반드시 `IDENTITY`로 설정해주세요.
2. PK 이름은 ERD대로 설정해주세요.